deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/ResourceLifeCycleHandler.java

```diff
@@ -95,6 +96,19 @@
     public List<T> getBySelector(ApiClient apiClient, String selector, boolean label, String namespace)
             throws HyscaleException;
 
+    /**
+     * Get all resources irrespective of namespace
+     * @param apiClient
+     * @param selector
+     * @param label
+     * @return List of resource
+     * @throws HyscaleException
+     */
+    default List<T> getForAllNamespaces(ApiClient apiClient, String selector, boolean label)
```

Comment:
listAllNamespaces ?

Reply:
listForAllNamespaces? on the lines with API method?

Reply:
yeah getForAllNamespaces is not making sense ? Can you think of a better name !